shared_model/backend/plain/impl/peer.cpp

```diff
@@ -9,7 +9,7 @@ using namespace shared_model;
 using namespace shared_model::plain;
 
 Peer::Peer(
-    const interface::types::AddressType &address,
+    const interface::types::AddressTypeView address,
```

Comment:
What is the reason to use this typedef. std::string_view looks more clear.

Reply:
agree, but the use of these types has historically been established.

Reply:
The reason to have strong types is to prevent passing an arbitrary string as address, in this case. Strong type requires the caller to explicitly create "AddressType".